service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.reform.idam.client.IdamApi;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.fpl.utils.ResourceReader.readBytes;
+
+@ExtendWith(SpringExtension.class)
+@WebMvcTest
+@OverrideAutoConfiguration(enabled = true)
+class CaseInitiationControllerTest {
+
+    private static final String AUTH_TOKEN = "Bearer token";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private IdamApi idamApi;
+
+    @Test
+    void shouldAddCaseLocalAuthorityToCaseData() throws Exception {
```

Comment:
Please add additional test case to assert that when user is not recognised endpoint returns no data block but error array with one error instead.

Reply:
Will this ever happen? A user needs to be on Idam to login, and their authorisation token will always link them to that entry in idam